### PR TITLE
Home page ready for review

### PR DESCRIPTION
**Pull request by MrSamich**
The circles in the background z-index are not yet added.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,21 +1,520 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=DM+Serif+Display:ital@0;1&family=Public+Sans:wght@400;700&display=auto"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./styles/tailwind.css" />
+    <script type="module" src="./scripts/main.js"></script>
+  </head>
+  <body class="bg-secondary-Isabelline">
+    <div
```
Comment (kiljaja) [inline index.html]: This element is showing navigation and should be a  `nav` element.

File: index.html [hunk 2]
```diff
@@ -1,21 +1,520 @@
...
+        </li>
+        <li class="hidden md:block pl-5 pr-10"><a href="">Pricing</a></li>
+        <li class="hidden md:block pr-10"><a href="">About</a></li>
+        <li class="hidden md:block"><a href="">Contact</a></li>
+      </ul>
+      <div class="hidden md:flex justify-end items-center">
+        <button class="btn btn--primary">Shedule a Demo</button>
+      </div>
+      <div class="md:hidden flex justify-end items-center">
+        <img src="./styles/assets/shared/mobile/menu.svg" alt="" />
```
Comment (kiljaja) [inline index.html]: The `assets` directory doesn't containing styling and need to be outside of the `styles` directory

File: index.html [hunk 3]
```diff
@@ -1,21 +1,520 @@
...
+            md:leading-14
+            laptop-large:leading-18
+            md:-tracking-0.092
+            laptop-large:-tracking-0.138
+            text-secondary-san-juan-blue
+          "
+        >
+          Start building with our APIs for absolutely free.
+        </p>
+        <div class="w-full md:flex md:items-end md:justify-end">
```
Comment (kiljaja) [inline index.html]: The `input` and the `button` should be inside a `form` element. The  `button` overflows outside of the input border in tablet and desktop view. It appears contained in the Figma sketch
Screen Shot 2021-11-22 at 8 00 25 PM

File: index.html [hunk 4]
```diff
@@ -1,21 +1,520 @@
...
+        laptop-large:flex-row
+        justify-center
+        items-center
+        px-6
+        md:px-24
+        pt-20
+        pb-10
+      "
+    >
+      <img
```
Comment (kiljaja) [inline index.html]: This `svg` is not showing code indentation.

expected visual
Screen Shot 2021-11-22 at 8 14 33 PM


current visual
Screen Shot 2021-11-22 at 8 12 28 PM

File: index.html [hunk 5]
```diff
@@ -1,21 +1,520 @@
...
+            w-full
+            md:w-max
+            text-center
+          "
+        >
+          Schedule a Demo
+        </button>
+      </div>
+    </div>
+    <div
```
Comment (kiljaja) [inline index.html]: This section is a footer and should use the `footer` element. The logo is not inline with the rest of the nav links
Screen Shot 2021-11-22 at 8 16 50 PM

**Review by kiljaja — CHANGES_REQUESTED** [review]
- Please refactor the page to use semantic elements for example `nav`, `form`, `footer`, and any others describes intention instead of `div` 

- Move `assets` directory out of the `styles` directory

- Some elements are not showing the drop shadow that is present in the Figma Sketch

File: index.html (hunk 1 above)
Comment (MrSamich) [reply index.html]: semantics have been adjusted.

File: index.html (hunk 2 above)
Comment (MrSamich) [reply index.html]: assets directory have been set outside the styles folder.

File: index.html (hunk 3 above)
Comment (MrSamich) [reply index.html]: size adjusted and box shadow added to input.

File: index.html (hunk 5 above)
Comment (MrSamich) [reply index.html]: footer nav properly centered.

File: index.html (hunk 4 above)
Comment (kiljaja) [reply index.html]: Closing because we can't find a reasonable solution.

**Pull request merged**